samples/radio_test/src/radio_test.h

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#ifndef RADIO_TEST_H_
+#define RADIO_TEST_H_
+
+#include <zephyr/types.h>
+#include <nrfx/hal/nrf_radio.h>
+
+/** Indicates devices that support BLE LR and 802.15.4 radio modes. */
+#define USE_MORE_RADIO_MODES (defined(NRF52840_XXAA) || \
+			      defined(NRF52833_XXAA) || \
+			      defined(NRF5340_XXAA_NETWORK))
+
+/** Maximum radio RX or TX payload. */
+#define RADIO_MAX_PAYLOAD_LEN	256
+/** IEEE 802.15.4 maximum payload length. */
+#define IEEE_MAX_PAYLOAD_LEN	127
+/** IEEE 802.15.4 minimum channel. */
+#define IEEE_MIN_CHANNEL	11
+/** IEEE 802.15.4 maximum channel. */
+#define IEEE_MAX_CHANNEL	26
+
+/**@brief Radio transmit and address pattern. */
+enum transmit_pattern {
+	/** Random pattern generated by RNG. */
+	TRANSMIT_PATTERN_RANDOM,
+
+	/** Pattern 11110000(F0). */
+	TRANSMIT_PATTERN_11110000,
+
+	/** Pattern 11001100(CC). */
+	TRANSMIT_PATTERN_11001100,
+};
+
+/**@brief Radio test configuration. */
+struct radio_test_config {
+	/** Radio transmission pattern. */
+	enum transmit_pattern tx_pattern;
+
+	/** Radio mode. Data rate and modulation. */
+	nrf_radio_mode_t mode;
+
+	/** Radio output power. */
+	u8_t txpower;
+
+	/** Radio start channel (frequency). */
+	u8_t channel_start;
+
+	/** Radio end channel (frequency). */
+	u8_t channel_end;
+
+	/* Delay time in milliseconds. */
```

Comment:
I think that it should be `/**` instead of  `/*` or vice versa. In case if you want to generate it in doc use `/**`. Please align it in all places